extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/ResourceExistenceChecks.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.jet.mongodb;
+
+import java.io.Serializable;
+
+/**
+ * Defines when and if database and collection existence will be checked.
+ * @since 5.4
+ */
+public enum ResourceExistenceChecks implements Serializable {
```

Comment:
`ResourceChecks` would still convey the meaning and would be less verbose.

Reply:
d365f2f8795dc76b6ecdab7884db53e094f7e06a